cfgov/v1/tests/test_filterable_list_form.py

```diff
@@ -31,7 +31,7 @@ def setUpTestData(cls):
         blog1.categories.add(CFGOVPageCategory(name='bar'))
         blog1.tags.add('foo')
         blog1.authors.add('richa-agarwal')
-        blog1.authors.add('sarah-simpson')
+        blog1.language.add('es')
```

Comment:
For model fields like `authors`, the `add()` method adds a related object to them. Because `language` is a field on the model, you could actually pass it in the constructor (`BlogPage(title='test page', language='es')`), or just use `blog1.language = 'es'`.